FrontMen/fm-hours
# Feat: create employee without login

# Changes

## Related issues

Resolves #61 

## Added

- Email field on employee schema
- "+ New employee" button on the employees' page allows admins to create new employees without the need for a previous login of the employee.

## Removed

- N/A

## Changed

- Now the Firebase Function is responsible to check for first-time logins and create new Employees.
- The Firebase/Nuxt auth hook is responsible to fetch the employee and update the vuex state.

## How to test

- Create a new employee manually with a google email that you have access
- Logout and try to sign in with the email that you just used to create the account
- The new account should be working correctly with the configuration set by the admin account, before the sign-in.

## Screenshots

![Screen Shot 2021-05-04 at 18 14 55-squashed](https://user-images.githubusercontent.com/45885054/117070932-cdd9a780-ad04-11eb-826e-c09684e8375c.png)

![Screen Shot 2021-05-04 at 18 15 07-squashed](https://user-images.githubusercontent.com/45885054/117070946-d336f200-ad04-11eb-863f-298a72a44987.png)
